src/hotspot/share/oops/constantPool.cpp

```diff
@@ -234,12 +234,12 @@ void ConstantPool::allocate_resolved_klasses(ClassLoaderData* loader_data, int n
 void ConstantPool::initialize_unresolved_klasses(ClassLoaderData* loader_data, TRAPS) {
   int len = length();
   int num_klasses = 0;
-  for (int i = 1; i <len; i++) {
-    switch (tag_at(i).value()) {
+  for (int cp_index = 1; cp_index <len; cp_index++) {
```

Comment:
I don't think the use of `i` as a simple loop variable caused any problem in understanding here.